src/components/molecules/ScheduleItemNG/ScheduleItemNG.scss

```diff
@@ -1,27 +1,102 @@
 @import "scss/constants.scss";
 
+$schedule--grey: opaque-white(0.7);
+$action-button-width: 136px;
+$bookmark-icon-size: 30px;
+$schedule-info-block-width: 60px;
+
 .ScheduleItemNG {
   display: flex;
-  padding-bottom: $ScheduleNG--padding;
+  padding: $spacing__lg;
 
-  &--info {
-    flex-basis: 25%;
-    margin-right: $ScheduleNG--padding;
+  &:hover {
+    background: linear-gradient(0deg, opaque-white(0.05), opaque-white(0.05));
+    border-radius: 12px;
   }
 
-  &--time,
-  &--name {
-    display: block;
-    font-weight: $font-weight--700;
+  &__info {
+    margin-top: $spacing--xs;
+    width: $schedule-info-block-width;
+
+    &--active {
+      margin-top: 0;
+      text-align: center;
+    }
+  }
+
+  &__details {
+    margin-left: $spacing--sm;
+  }
+
+  &__buttons {
+    display: flex;
+  }
+
+  &__button {
+    width: $action-button-width;
+    padding: $spacing--sm;
+    font-weight: $font-weight--500;
+    font-size: $font-size--md;
+    line-height: 17px;
```

Comment:
not sure if it's right to add some specific vars for `ScduleItemNG` elemnt's line-heights, like `15px` `17px` or `18px`. let's leave it like `line-height: 17px;`

Reply:
across the codebase we don't seem to use variables for line-height. maybe this could be done with a variable in constants.scss and used in many places in a separate pr?

Reply:
i meant in-file variable. no need to create a separate const in constants.scss. but if you both think it's not needed, it's ok.